package.json

```diff
@@ -67,5 +67,9 @@
     "files": [
         "dist/",
         "Dynamo"
-    ]
+    ],
+    "engines": {
+        "node": ">=12.0.0",
```

Comment:
@mobsense; I expect lots of people are using this library with node 12, but I believe from the docs certain features only work with node 14. 

Should I make this greater than or equal to node 14?

Reply:
I can't remember being dependent on node 12. We do require NPM 7 if you want to use AWS SDK V3.

Because we transpile, we should not be requiring node 14.

Reply:
Ok super, my guess would be that it makes sense to add node 12.x to the matrix of tests?

Here is a link to my fork where I have made that change: https://github.com/simonireilly/dynamodb-onetable/pull/1

That one could go in separately as well? What do you think?

Reply:
Yes, makes sense.